packages/clerk-js/src/ui/styledSystem/animations.ts

```diff
@@ -84,7 +84,7 @@ const notificationAnimation = keyframes`
 const outAnimation = keyframes`
   0% {
     opacity:1;
-    translateY(0px);
+    translateY: (0px);
```

Comment:
Actually we need this, right ?
```suggestion
  transform: translateY(0px)
```

Reply:
Right! It was missing the translate property